client/README.md

```diff
@@ -0,0 +1,34 @@
+# Vue.js UI for [HoldMail](https://github.com/spartasystems/holdmail)
+This project is used as an educational tool for understanding the Vue.js ecosystem. This project replicates all of the UI features that HoldMail (using Angular) has today. This project is meant to compare UI technologies and learn more about [Vue.js](https://vuejs.org/v2/guide).
```

Comment:
Will need to update this since it's no longer a 'hey-let's-look-at-vue' effort for us :)  I'll also make sure to update all READMEs as part of the prep for the next release.

Reply:
Will do.